web-services-app/src/test/java/edu/unc/lib/boxc/web/services/utils/ExporterUtil.java

```diff
@@ -0,0 +1,44 @@
+package edu.unc.lib.boxc.web.services.utils;
+
+import edu.unc.lib.boxc.search.api.models.ContentObjectRecord;
+import edu.unc.lib.boxc.search.solr.responses.SearchResultResponse;
+import edu.unc.lib.boxc.search.solr.services.SolrSearchService;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+public class ExporterUtil {
```

Comment:
I'd suggest naming this something like ExporterTestUtil to make it clear that it is only used in testing, since that's not obvious at the moment when it's being used in the tests.

Also, we should add javadoc to this class